src/main/java/ua/com/juja/microservices/teams/slackbot/repository/impl/FakeUserRepository.java

```diff
@@ -0,0 +1,99 @@
+package ua.com.juja.microservices.teams.slackbot.repository.impl;
```

Comment:
Impossible to run application with bean in test package. Spring don't find bean there. 
java -jar -Dspring.profiles.active=test teams-slack-bot.jar